content/sensu-go/5.14/guides/clustering.md

```diff
@@ -0,0 +1,269 @@
+
+### Traditional computer instance
+
+_NOTE: The remainder of this guide uses on disk configuration. If you are using an ephemeral computer instance, you can use `sensu-backend start --help` to see examples of etcd command line flags. The configuration file entries below translate to `sensu-backend` flags._
+
+#### Sensu backend configuration
+
+Below are example configuration snippets from `/etc/sensu/backend.yml` using a three node cluster. The nodes are named `backend-1`, `backend-2` and `backend-3` with IP addresses `10.0.0.1`, `10.0.0.2` and `10.0.0.3`, respectively.
+
+_NOTE: This backend configuration assumes you have set up and installed the sensu-backend on all the nodes used in your cluster. You can use our [installation and configuration guide][14] guide if you have not done so._
+
+**backend-1**
+
+{{< highlight yml >}}
+##
+# store configuration for backend-1/10.0.0.1
+##
+ etcd-advertise-client-urls: "http://10.0.0.1:2379"
+etcd-listen-client-urls: "http://10.0.0.1:2379"
+etcd-listen-peer-urls: "http://0.0.0.0:2380"
+etcd-initial-cluster: "backend-1=http://10.0.0.1:2380,backend-2=http://10.0.0.2:2380,backend-3=http://10.0.0.3:2380"
+etcd-initial-advertise-peer-urls: "http://10.0.0.1:2380"
+etcd-initial-cluster-state: "new"
+etcd-initial-cluster-token: ""
+etcd-name: "backend-1"
+{{< /highlight >}}
+
+**backend-2**
+
+{{< highlight yml >}}
+##
+# store configuration for backend-2/10.0.0.2
+##
+etcd-advertise-client-urls: "http://10.0.0.2:2379"
+etcd-listen-client-urls: "http://10.0.0.2:2379"
+etcd-listen-peer-urls: "http://0.0.0.0:2380"
+etcd-initial-cluster: "backend-1=http://10.0.0.1:2380,backend-2=http://10.0.0.2:2380,backend-3=http://10.0.0.3:2380"
+etcd-initial-advertise-peer-urls: "http://10.0.0.2:2380"
+etcd-initial-cluster-state: "new"
+etcd-initial-cluster-token: ""
+etcd-name: "backend-2"
+{{< /highlight >}}
+
+**backend-3**
+
+{{< highlight yml >}}
+##
+# store configuration for backend-3/10.0.0.3
+##
+etcd-advertise-client-urls: "http://10.0.0.3:2379"
+etcd-listen-client-urls: "http://10.0.0.3:2379"
+etcd-listen-peer-urls: "http://0.0.0.0:2380"
+etcd-initial-cluster: "backend-1=http://10.0.0.1:2380,backend-2=http://10.0.0.2:2380,backend-3=http://10.0.0.3:2380"
+etcd-initial-advertise-peer-urls: "http://10.0.0.3:2380"
+etcd-initial-cluster-state: "new"
+etcd-initial-cluster-token: ""
+etcd-name: "backend-3"
+{{< /highlight >}}
+
+Once each node has the configuration described above, start each sensu-backend:
+
+{{< highlight shell >}}
+sudo systemctl start sensu-backend
+{{< /highlight >}}
+
+#### Adding sensu agents to the cluster
+
+Each Sensu agent should have the following entries in `/etc/sensu/agent.yml` to ensure they are aware of all cluster members. This allows the agent to reconnect to a working backend in the scenrio where the one it is currently connected to goes into an unhealthy state.
```

Comment:
```suggestion
Each Sensu agent should have the following entries in `/etc/sensu/agent.yml` to ensure they are aware of all cluster members. This allows the agent to reconnect to a working backend if the cluster member it is currently connected to goes into an unhealthy state.
```

Reply:
Please confirm that "cluster member" is what you meant by "one" in the second sentence here.